release-notes/May_2016.md

```diff
@@ -11,12 +11,12 @@ The May release of VS Code has some great new features, enhancements to existing
 
 Here are the highlights:
 
-* **Editor**: Ability to trim automatically inserted whitespace, a new key binding to select all find matches, and the Peek view is now resizable.
-* **Linting**: ESLint now supports the ability to fix all problems and TSLint supports quick fixes and file exclusion.
-* **Languages**: Code completion (IntelliSense), Go To and Peek Definition, and Find All Occurrences support for CSS variables in CSS, SCSS, and LESS files.
-* **Workbench**: A new integrated Terminal allows you to stay in VS Code while using your platform's shell, we've added the ability to remove key bindings, and we've made progress on Tab support.
-* **Debugging**: You can now control how the Debug Console is revealed and we've improved stepping performance for Node.js applications with large numbers of local variables.
-* **Extensions**: You can now list, install, and uninstall extensions from the command line. For extension authors, we've expanded our debug, TextMate grammar, and command APIs and we've improved the Language Server Protocol.
+* [**Editor**](#_editor): Ability to trim automatically inserted whitespace, a new key binding to select all find matches, and the Peek view is now resizable.

```

Comment:
If you want the links to work in GitHub, you can give the full path ex. /releasenotes/May_2016.md#editor).  We then convert to the right path for web navigation when we compile the Markdown.